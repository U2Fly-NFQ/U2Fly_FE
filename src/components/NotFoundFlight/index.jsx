import LoadingGif from '../../assets/images/system/loading-animated.gif'

const NotFoundFlight = () => {
  return (
    <img
      style={{
        display: 'block',
        margin: 'auto',
        maxWidth: '500px',
        objectFit: 'contain',
      }}
      src={LoadingGif}
      alt="Loading GIF"
    />
  )
}

export default NotFoundFlight
