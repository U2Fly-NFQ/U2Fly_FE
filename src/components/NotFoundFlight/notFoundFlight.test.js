import renderer from 'react-test-renderer'
import React from 'react'
import NotFoundFlight from './'

describe('Not found page test', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NotFoundFlight />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
