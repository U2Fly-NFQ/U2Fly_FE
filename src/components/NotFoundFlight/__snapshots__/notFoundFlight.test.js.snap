// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page test renders correctly 1`] = `
<div
  className="not-found-flight"
>
  <div
    className="ant-row"
    style={Object {}}
  >
    <div
      className="ant-col ant-col-24"
      justify="center"
      style={Object {}}
    >
      <span
        className="ant-typography"
        onClick={null}
        style={
          Object {
            "WebkitLineClamp": undefined,
            "display": "block",
            "textAlign": "center",
          }
        }
      >
        Sorry we didn't find any results matching this search
      </span>
      <img
        alt="Loading GIF"
        src="not-found.webp"
        style={
          Object {
            "display": "block",
            "margin": "auto",
            "maxHeight": "300px",
            "maxWidth": "100%",
            "objectFit": "contain",
          }
        }
      />
    </div>
  </div>
</div>
`;
